config/migrations/working-time-category-concept-scheme-duplicate.sparql

```diff
@@ -0,0 +1,20 @@
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/application> {
+<http://lblod.data.gift/concept-schemes/95d22e7f-de86-4a2c-9974-0e3de5013d87> <http://mu.semte.ch/vocabularies/core/uuid> "95d22e7f-de86-4a2c-9974-0e3de5013d87" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#ConceptScheme> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Working time category"@en, "Werktijd categorie"@nl .
```

Comment:
Which one of the `Working time category concept-scheme` do we keep? Or how do we update the naming of it that this is non conflicting to select?